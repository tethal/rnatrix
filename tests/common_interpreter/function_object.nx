fun f(a) {
    return a + 1;
}

fun a() {
    return f;
}

fun b(x, y) {
    return x(y);
}

fun main() {
    var x = a();
    print(b(x, 42));
    print(str);
}
