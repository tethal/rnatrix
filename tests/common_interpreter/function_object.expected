43
<built-in function str>
